app/controllers/ProjectSidewalkAPIController.scala

```diff
@@ -89,18 +99,29 @@ class ProjectSidewalkAPIController @Inject()(implicit val env: Environment[User,
     * @param severity
     * @return
     */
-  def getAccessAttributesV2(lat1: Double, lng1: Double, lat2: Double, lng2: Double, severity: Option[String]) = UserAwareAction.async { implicit request =>
+  def getAccessAttributesV2(lat1: Double, lng1: Double, lat2: Double, lng2: Double, severity: Option[String], filetype: Option[String]) = UserAwareAction.async { implicit request =>
     apiLogging(request.remoteAddress, request.identity, request.toString)
 
     val minLat:Float = min(lat1, lat2).toFloat
     val maxLat:Float = max(lat1, lat2).toFloat
     val minLng:Float = min(lng1, lng2).toFloat
     val maxLng:Float = max(lng1, lng2).toFloat
 
-    val features: List[JsObject] =
-      GlobalAttributeTable.getGlobalAttributesInBoundingBox(minLat, minLng, maxLat, maxLng, severity).map(_.toJSON)
-
-    Future.successful(Ok(Json.obj("type" -> "FeatureCollection", "features" -> features)))
+    if(filetype != None && filetype.get == "csv") {  // CSV format
+      val AccessAttributesfile = new java.io.File("access_attributes.csv")
```

Comment:
I had a strange issue here where I had the variable named file and it was deleting the file before it sent it to the user, so it was throwing an error.  It's weird because for all of the other http responses using the same line with the onClose function it works

Reply:
Very weird... Well as long as the variable name doesn't start with a capital letter it's fine :+1: